Sources/Yams/Emitter.swift

```diff
@@ -176,25 +183,29 @@ public func serialize(
         explicitStart: explicitStart,
         explicitEnd: explicitEnd,
         version: version,
-        sortKeys: sortKeys)
+        sortKeys: sortKeys
+    )
 }
 
+/// Class responsible for emitting libYAML events.
 public final class Emitter {
+    /// Line break options to use when emitting YAML.
     public enum LineBreak {
         /// Use CR for line breaks (Mac style).
-        case cr // swiftlint:disable:this identifier_name
+        case cr
         /// Use LN for line breaks (Unix style).
-        case ln // swiftlint:disable:this identifier_name
+        case ln
         /// Use CR LN for line breaks (DOS style).
         case crln
     }
 
-    public var data = Data()
+    internal var data = Data()
```

Comment:
It would be `public internal(set)` since this is the only way to get `Emitter` 's output, so if we do not publish it, it makes no sense to publish `Emitter`.